runners/direct-java/build.gradle

```diff
@@ -178,6 +192,26 @@ task validatesRunner(type: Test) {
   }
 }
 
+tasks.register("validatesRunnerSickbay", Test) {
```

Comment:
This task gets duplicated a lot, with the only differences being the runner and the classpath (and maybe the sickbayed test?). Instead of duplicating the code, could you just programatically create them like this? https://github.com/apache/beam/blob/e6c73b40f8376ec5088f69360108200ee5bd3813/examples/java/twitter/build.gradle#L81

Reply:
Hi Danny! I was creating the tasks programmatically, but since the tests for each runner are different, wouldn't it be better to leave the tasks like this instead of using conditional statements or creating a whole class hierarchy to dynamically access the tests for each runner?
Sorry if this a silly question.

Reply:
Yeah, actually taking another look I think you're right that this is more trouble than its worth, especially since we also need the tests in needsRunnerTests so there's not a super clean way to pull them out. Thanks for pushing on this.